ThunderCloud/QuizQuestionContainerViewController.swift

```diff
@@ -136,6 +136,16 @@ class QuizQuestionContainerViewController: UIViewController {
 		hintLabel.isHidden = question?.hint == nil
 		hintLabel.text = question?.hint
 	}
+    
+    override func viewWillDisappear(_ animated: Bool) {
+        super.viewWillDisappear(animated)
+        navigationController?.interactivePopGestureRecognizer?.delegate = nil;
+    }
+    
+    override func viewDidAppear(_ animated: Bool) {
+        super.viewDidAppear(animated)
+        navigationController?.interactivePopGestureRecognizer?.delegate = self;
```

Comment:
It'd be awesome to have this documented, just so it's super clear in the future why this is here :)